app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt

```diff
@@ -8,7 +8,7 @@ import org.simple.clinic.mobius.next
 class MedicineFrequencyUpdate : Update<MedicineFrequencyModel, MedicineFrequencyEvent, MedicineFrequencyEffect> {
   override fun update(model: MedicineFrequencyModel, event: MedicineFrequencyEvent): Next<MedicineFrequencyModel, MedicineFrequencyEffect> {
     return when (event) {
-      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(event.medicineFrequency))
+      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(model.medicineFrequency))
```

Comment:
With this, we can also remove the `medicineFrequency` property from the `SaveMedicineFrequencyClicked` class, right?

Reply:
Fair enough, I'll update it